From 4f64adf37d5108574ac8db6bd60486b30c6508b7 Mon Sep 17 00:00:00 2001
From: matt <matthilton2005@gmail.com>
Date: Fri, 16 Jul 2021 23:32:35 +0100
Subject: [PATCH] Added QTDbus fix

---
 plugins/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index 41dce58..1f7fb48 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -24,6 +24,7 @@ FOREACH(plugin_type ${plugin_types})
             ${Qt5MultimediaWidgets_INCLUDE_DIRS}
             ${Qt5Widgets_INCLUDE_DIRS}
             ${Qt5Bluetooth_INCLUDE_DIRS}
+            ${Qt5DBus_INCLUDE_DIRS}
             ${Boost_INCLUDE_DIRS}
             ${LIBUSB_1_INCLUDE_DIRS}
             ${PROTOBUF_INCLUDE_DIR}
-- 
2.32.0.windows.1

