From 8d1f16591c186830146c95d8e96d4830ee4fbca8 Mon Sep 17 00:00:00 2001
From: matt <matthilton2005@gmail.com>
Date: Sun, 18 Jul 2021 21:53:16 +0100
Subject: [PATCH] Added fix for QtDBus error to fix #73

---
 CMakeLists.txt         | 4 +++-
 plugins/CMakeLists.txt | 2 ++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c7583c9..e466dd3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,7 +34,7 @@ add_definitions(-DBOOST_ALL_DYN_LINK)
 
 find_package(Boost REQUIRED COMPONENTS system log OPTIONAL_COMPONENTS unit_test_framework)
 find_package(libusb-1.0 REQUIRED)
-find_package(Qt5 COMPONENTS Multimedia MultimediaWidgets Bluetooth Qml Quick QuickWidgets SerialBus SerialPort WebSockets REQUIRED)
+find_package(Qt5 COMPONENTS Multimedia MultimediaWidgets Bluetooth Qml Quick QuickWidgets SerialBus SerialPort WebSockets DBus REQUIRED)
 find_package(KF5BluezQt REQUIRED)
 find_package(Protobuf REQUIRED)
 find_package(OpenSSL REQUIRED)
@@ -92,6 +92,7 @@ target_include_directories(dash PRIVATE
 
 target_include_directories(dash SYSTEM PRIVATE
     ${Qt5WebSockets_INCLUDE_DIRS}
+    ${Qt5DBus_INCLUDE_DIRS}
     ${Qt5SerialBus_INCLUDE_DIRS}
     ${Qt5SerialPort_INCLUDE_DIRS}
     ${Qt5Multimedia_INCLUDE_DIRS}
@@ -119,6 +120,7 @@ target_link_libraries(dash
     aasdk
     openauto
     ${Boost_LIBRARIES}
+    ${Qt5DBus_LIBRARIES}
     ${Qt5WebSockets_LIBRARIES}
     ${Qt5SerialBus_LIBRARIES}
     ${Qt5SerialPort_LIBRARIES}
diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index 41dce58..c6253fe 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -24,6 +24,7 @@ FOREACH(plugin_type ${plugin_types})
             ${Qt5MultimediaWidgets_INCLUDE_DIRS}
             ${Qt5Widgets_INCLUDE_DIRS}
             ${Qt5Bluetooth_INCLUDE_DIRS}
+            ${Qt5DBus_INCLUDE_DIRS}
             ${Boost_INCLUDE_DIRS}
             ${LIBUSB_1_INCLUDE_DIRS}
             ${PROTOBUF_INCLUDE_DIR}
@@ -44,6 +45,7 @@ FOREACH(plugin_type ${plugin_types})
             aasdk_proto
             aasdk
             openauto
+            ${Qt5DBus_LIBRARIES}
             ${Boost_LIBRARIES}
             ${Qt5WebSockets_LIBRARIES}
             ${Qt5SerialBus_LIBRARIES}
-- 
2.32.0.windows.1

