From 4b8af7fd81ab1fdb20b2e0c778194bb435bc406f Mon Sep 17 00:00:00 2001
From: matt <matthilton2005@gmail.com>
Date: Sun, 18 Jul 2021 21:31:42 +0100
Subject: [PATCH] Dbus patch

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a8e297c..4344978 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -93,6 +93,7 @@ target_include_directories(dash PRIVATE
 
 target_include_directories(dash SYSTEM PRIVATE
     ${Qt5WebSockets_INCLUDE_DIRS}
+    ${Qt5DBus_INCLUDE_DIRS}
     ${Qt5SerialBus_INCLUDE_DIRS}
     ${Qt5SerialPort_INCLUDE_DIRS}
     ${Qt5Multimedia_INCLUDE_DIRS}
@@ -120,6 +121,7 @@ target_link_libraries(dash
     aasdk
     openauto
     ${Boost_LIBRARIES}
+    ${Qt5DBus_LIBRARIES}
     ${Qt5WebSockets_LIBRARIES}
     ${Qt5SerialBus_LIBRARIES}
     ${Qt5SerialPort_LIBRARIES}
-- 
2.32.0.windows.1

